src/Billing/index.js

```diff
@@ -0,0 +1,90 @@
+import { unpackData } from '../utils'
+
+/**
+ * Billing API Client
+ */
+export default class Billing {
+  /**
+   * Create new Client for the Billing API
+   * @params {Client} client - Scalingo API Client
+   */
+  constructor(client) {
+    this._client = client
+  }
+
+  /**
+   * Return the billing profile of current user
+   * @see https://developers.scalingo.com/billing#billing-profile
+   * @return {Promise<BillingProfile | APIError>}
+   */
+  profile() {
+    return unpackData(
+      this._client.billingApiClient().get('/profile'),
+      'profile',
```

Comment:
```suggestion
      'billing_profile',
```
The dev doc says that the key of the response is `billing_profile`

Reply:
The dev docs refer to the api.scalingo endpoint, not the billing api's ones; those are different and not documented yet. I've relied on having access to the source of the api to find the appropriate interface.

Reply:
Arf OK, sorry